makefile

```diff
@@ -107,7 +107,7 @@ e2e-test: prepare
 	sudo sysctl -w vm.nr_hugepages=0
 
 	@echo "=====Running e2e tests"
-	$(if $(saveOutput),gotestsum --junitfile test/test_results/e2e-tests.xml ./test/e2e/... -count=1 -timeout 70m, go test -v -count=1 -timeout 70m ./test/e2e/...)
+	$(if $(saveOutput),gotestsum --junitfile test/test_results/e2e-tests.xml ./test/e2e/tls_test.go -count=1 -timeout 70m, go test -v -count=1 -timeout 70m ./test/e2e/tls_test.go)
```

Comment:
this change should be reverted to run all tests in the e2e directory.

Reply:
I am doing debug with only one test